content/docs/start/data-and-model-access.md

```diff
@@ -2,29 +2,29 @@
 title: 'Get Started: Data and Model Access'
 ---
 
-# Get Started: Data and Model Access
+# Data and Model Access
 
-Okay, we've learned how to _track_ data and models with DVC, and how to commit
-their versions to Git. The next questions are: How can we _use_ these artifacts
-outside of the project? How do I download a model to deploy it? How to download
+We've learned how to _track_ data and models with DVC, and how to commit their
+versions to Git. The next questions are: How can we _use_ these artifacts
+outside of the project? How do we download a model to deploy it? How to download
 a specific version of a model? Or reuse datasets across different projects?
 
-> These questions tend to come up when you browse the files that DVC saves to
-> remote storage, e.g.
+> These questions tend to come up when you browse the files which DVC saves to
+> remote storage (e.g.
```

Comment:
On `,` -> `(` it's good that you read and review all this very closely but does it really make a difference? 🙂 I don't mind either way, I'd just try to avoid restyling existing text (no point probably: we'll always have slightly different styles per author anyway).

Reply:
this isn't stylistic - this is purely about readability. "Which" is more precise, and the parentheses in this case demarcate an aside, replacing a comma splice. Technically the comma could've been replaced with a semicolon instead.

Reply:
"Which" doesn't sound correct here. As I mentioned, the clause is restrictive https://www.merriam-webster.com/words-at-play/when-to-use-that-and-which

Also, `, e.g.` is very common, at least in US academia in my experience. See https://english.stackexchange.com/questions/415616/proper-placement-of-e-g nothing wrong with it.

Please let's not focus on style. If there are big obvious non subjective errors in writing then absolutely let's fix them when we see them, otherwise we're going to spend most of our time arguing grammatical forms and we're not a publisher.

Reply:
> "Which" doesn't sound correct here.

Ah MW frequently riles by UK-centric sensibilities (and also note that the MW link itself acknowledges that "the best writers seem to disregard" the rule). I follow a different rule.

However I think consistency is paramount as since the entirety of the docs is very Americaneseishy I'm happy to stick to MW.

But... I think MW is consistent with my edit in this case. The MW article's conclusion is unrestricted -> which; restricted -> personal style choice.

This particular sentence is bordering on unrestricted so really "which" is technically safest in all cases :)

> Also, `, e.g.` is very common

Nothing wrong with `, e.g.` - rather I was placing the entire following sentiment into parens.

Anyway I think it's worth on occasion debating nuances lest future docs suffer. It's not about this PR, but about the millions to come.

Reply:
> "the best writers seem to disregard" the rule.) I follow a different rule.

Sure. Just again, no strong reason to change, it's subjective, etc. I don't have a strong opinion on the specific text. I just think the best route here is to let docs have different styles per author, it's prob OK and mainly we don't have the capacity to have this kind of discussion super often but def appreciate it now and then as you mentioned 🙂 

> consistency is paramount

That too! But on the other hand if you go for new docs next don't worry about sounding Americaneseishy 🇬🇧 

> restricted -> personal style choice

That would mean leave the existing style IMO. But again whatevs, let's move on.